modulus/launch/logging/launch.py

```diff
@@ -324,6 +325,51 @@ def _log_backends(
             metric_dict[step[0]] = step[1]
             wandb.log(metric_dict)
 
+    def log_figure(
+        self,
+        figure,
```

Comment:
FYI removed type hint so matplotlib is not a requirement

Reply:
Instead of removing, you could do something like `figure : "Figure"` though this may or may not upset type checker.